code/src/components/Like.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import './like.css'
+
+export const Like = ({ onLiked, heart, heartId }) => {
+  const handleClick = () => {
+    fetch(
+      `https://happy-thoughts-technigo.herokuapp.com/thoughts/${heartId}/like`,
+      {
+        method: "POST",
+        body: "",
```

Comment:
No need for a body in this request, all essential informations are being passed in url in line 8